source/gameengine/Rasterizer/RAS_MeshObject.h

```diff
@@ -39,6 +39,7 @@
 
```

Comment:
Please avoid add python dependencies in the rasterizer part. Part allowed to use python are Ketsji, GameLogic, VideoTexture, Converter, and Launcher (in some time).